### PR TITLE
feat(Tooltip): use portals for tooltips

### PR DESCRIPTION
**Pull request by ajkl2533**
Closes FEP-1623

### CODE REVIEW

File: src/components/Tooltip/hooks/useTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  useEffect(() => {
+    setStyle(getPlacementStyles());
+  }, [getPlacementStyles, isShow]);
+
+  useEffect(() => {
+    scrollListener.current = () => {
+      hide();
+    };
+    if (isListenersAdded.current && isNotNull(scrollListener.current)) {
```
Comment (PrincAm) [inline src/components/Tooltip/hooks/useTooltip.tsx:35]: Pls, just check `isNotNull(scrollListener.current)` is not enough? Wondering if `isListenersAdded` ref is needed or what use case I'm missing

**Review by PrincAm — APPROVED** [review]
Nice. GJ!

File: src/components/Tooltip/hooks/useTooltip.tsx (hunk 1 above)
Comment (ajkl2533) [reply src/components/Tooltip/hooks/useTooltip.tsx:35]: The problem is we want to remove event listener from window, before we add another one, but if you try to remove event listener which doesn't exist it throws. So this we we check if there is any previous listener which we can remove, if not we just skip the removing part

Comment (PrincAm) [reply src/components/Tooltip/hooks/useTooltip.tsx:35]: Check of previous listener, got it

**Pull request merged**